qutebrowser/completion/models/sortfilter.py

```diff
@@ -146,10 +148,44 @@ def filterAcceptsRow(self, row, parent):
                 data = self.srcmodel.data(idx)
                 if not data:
                     continue
-                elif self.pattern.casefold() in data.casefold():
-                    return True
+                else:
+                    matchers = {
+                        'contain': self.containMatcher,
+                        'fuzzy': self.fuzzyMatcher,
+                        'start': self.startMatcher
```

Comment:
Please call those `_match_contains`, `_match_fuzzy` and `_match_start`. Rationales:

* "matcher" makes it sound like they are some kind of object, while it's just a method.
* `mixedCase` is only used for Qt's methods in qutebrowser, `snake_case` is used otherwise.
* Those aren't needed from outside this class, so they can be "private" (starting with `_`)